pkg/image/api/helper.go

```diff
@@ -474,6 +474,8 @@ func ImageWithMetadata(image *Image) error {
 			image.DockerImageLayers[i].Name = layer.DockerBlobSum
 		}
 		if len(manifest.History) == len(image.DockerImageLayers) {
+			// This code does not work since hub.docker.com has been updated.
+			// The V1Compatibility no longer contains Size.
```

Comment:
This will be true for docker hub, but for other repos supporting V1, it's not. I'd drop this comment entirely.